aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -164,6 +164,7 @@ Tensor& div_sparse_(Tensor& self, const Tensor& value) {
 SparseTensor& div_out_sparse_zerodim(SparseTensor& r, const SparseTensor& t, const Tensor& value) {
   TORCH_CHECK(value.dim() == 0, "sparse division only supports division by a scalar (got shape ",
       value.sizes(), " for argument 'other')");
+  TORCH_CHECK(!value.is_sparse(), "Unsupported tensor layout");
```

Comment:
It's good to tell the user what exactly is unsupported and what is unsupported. Judging by this, it seems like "sparse / sparse" is unsupported, but "sparse / dense" is supported. Is that right?

Also, this needs a test

Reply:
added, though I feel a bit silly adding a test just to validate the particular text of an error.

Reply:
Well, someone could come along and delete the error message and we'd never know. I think that having tests to validate that error messages exist and are what we expect is valuable to have.